pkg/publicapi/apimodels/node.go

```diff
@@ -67,10 +68,12 @@ func (n NodeAction) Description() string {
 		return "Approve a node whose membership is pending"
 	case NodeActionReject:
 		return "Reject a node whose membership is pending"
+	case NodeActionDelete:
+		return "Delete a node from the cluster. Note this does not stop them re-connecting."
```

Comment:
I don't think we need this warning. If a node disconnects, does it have to go pending again? (it should)

Reply:
Yes, would go back to Pending, unless whatever criteria we're using for auto-approve (tbd) puts them in that state.